examples/app-router/app/api/after/revalidate/route.ts

```diff
@@ -6,7 +6,8 @@ export function POST() {
     () =>
       new Promise<void>((resolve) =>
         setTimeout(() => {
-          revalidateTag("date");
+          // We want to expire the "date" tag immediately
+          revalidateTag("date", { expire: 0 });
```

Comment:
nit to make things clearer:

```suggestion          
          revalidateTag("date", { 
            // We want to expire the "date" tag immediately
            expire: 0 
          });
```